docs/OPTIONS-LIST.md

```diff
@@ -379,6 +377,14 @@ see [notes E-](/docs/NOTES.md#e-) for details
 
 ##         B2) GENERIC GENERAL/RANKED/UNRANKED ARGUMENTS :
 
+#### only (part 2)
+
+  ```--proonly``` For all matches, only accept those from professionals
+
+  ```--proonlyranked``` For ranked games, only accept those from professionals
+
+  ```--proonlyunranked``` For ranked games, only accept those from professionals
```

Comment:
Should say "For unranked games"?

Reply:
@wonderingabout Don't forget this comment

Reply:
yes, it's a mistake indeed

Reply:
fixed